Docs/landing/static/versions.json

```diff
@@ -1 +1 @@
-[{"version": "2.7"},{"version": "2.6"},{"version": "2.5"},{"version": "2.4"},{"version": "2.3"},{"version": "2.2"},{"version": "2.1"},{"version": "2.0"}, {"version": "1.11"}]
+[{"version": "2.8"},{"version": "2.7"},{"version": "2.6"},{"version": "2.5"},{"version": "2.4"},{"version": "2.3"},{"version": "2.2"},{"version": "2.1"},{"version": "2.0"}, {"version": "1.11"}]
```

Comment:
You can do a version per line if you don't want to keep making this line longer and longer.

Reply:
Done